plugins/rtmp-services/data/services.json

```diff
@@ -259,6 +259,40 @@
                 "max audio bitrate": 160
             }
         },
+	{
+            "name": "Loola.tv",
+            "common": false,
+            "servers": [             
+                {
+                    "name": "US East: Virginia",
+                    "url": "rtmp://rtmp.loola.tv/push"
+                },
+		{
+                    "name": "EU Central: Germany",
+                    "url": "rtmp://rtmp-eu.loola.tv/push"
+                },
+		{
+                    "name": "South America: Brazil",
+                    "url": "rtmp://rtmp-sa.loola.tv/push"
+                },
+		{
+                    "name": "Asia/Pacific: Singapore",
+                    "url": "rtmp://rtmp-sg.loola.tv/push"
+                },               
+                {
+                    "name": "Middle East: Bahrain",
+                    "url": "rtmp://rtmp-me.loola.tv/push"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "main",
+                "max video bitrate": 2500,
+                "max audio bitrate": 160,
+                "bframes": 2,
+                "x264opts": "scenecut=0 tune=zerolatency"
```

Comment:
Are you sure you want tune=zerolatency? This will greatly reduce quality of the encoder, for very minimal savings in terms of latency.

Reply:
I've measured a fairly significant 1 second improvement (veryfast, 720p30 @ 2500 kbps), which remained after setting bframes = 2.